address errataum 7888 (#61)
lamps-wg/cmp-updates: draft-ietf-lamps-rfc4210bis.md

### Initialization Response
{: id="sect-5.3.2"}

> An Initialization response message contains as the PKIBody a
> CertRepMessage data structure, which has for each certificate

Author: ```suggestion
An Initialization response message contains a
CertRepMessage data structure in PKIBody, which has for each certificate
```

Reviewer: The fixing of the erratum  (an -> a) here is fine.

But, PKIBody is a choice and I am wondering if 'as' is correct.
But I am not a native speaker, so I will follow your advice :-)
If we change the text here, we also need to change it in Sections 5.3.1, 5.3.3, and 5.3.4.

Reviewer: @seanturner should I only change the a -> an or should I also implement the change you proposed to Section  5.3.1, 5.3.3, and 5.3.4?
